assets/js/components/ClusterDetails/HanaClusterDetails.test.jsx

```diff
@@ -181,4 +182,51 @@ describe('HanaClusterDetails component', () => {
     expect(sidContainer).toHaveTextContent(sid);
     expect(sidContainer.querySelector('a')).toBeNull();
   });
+
+  it('should display a host link in the site details if the host is registered', () => {
+    const registeredClusterNode = hanaClusterDetailsNodesFactory.build({
+      name: 'registeredhost',
```

Comment:
Do we need this fixes name?
The factory will already create a name. `registeredClusterNode.name`